README.md

```diff
@@ -63,11 +63,12 @@ Producer
 * `USER_CRT` - the user's certificate
 * `USER_KEY` - the user's private key
 * `LOG_LEVEL` - logging level  
-* `PRODUCER_ACKS` = acknowledgement level
-* `HEADERS` = custom headers list separated by commas of `key1=value1, key2=value2`
-* `ADDITIONAL_CONFIG` = additional configuration for a producer application. The form is `key=value` records separated by new line character
-* `BLOCKING_PRODUCER` = if it's set, the producer will block another message until ack will be received
-* `MESSAGES_PER_TRANSACTION` = how many messages will be part of one transaction. Transaction config could be set via `ADDITIONAL_CONFIG` variable. Default is 10.
+* `PRODUCER_ACKS` - acknowledgement level
+* `HEADERS` - custom headers list separated by commas of `key1=value1, key2=value2`
+* `BLOCKING_PRODUCER` - if it's set, the producer will block another message until ack will be received
+* `MESSAGES_PER_TRANSACTION` - how many messages will be part of one transaction. Transaction config could be set via `ADDITIONAL_CONFIG` variable. Default is 10.
+* `SASL_CALLBACK_CLASS` - sets class for `sasl.login.callback.handler.class`; by default it uses `io.strimzi.kafka.oauth.client.JaasClientOauthLoginCallbackHandler` if possible
```

Comment:
out of curiosity ... if we are going to add this useful `ADDITIONAL_CONFIG`, why adding a new env var for SASL callback class instead of relying on the  `ADDITIONAL_CONFIG` itself?

Reply:
yeah, that's good point. I was thinking about it and forgot to change it. It makes no sense to have it. (maybe only for documentation purposes :) )

Reply:
Fixed (removed). Had to execute bunch of tests to make sure my change won't break Oauth tests.

Reply:
JFYI `ADDITIONAL_CONFIG` is part of example clients for quite a long time.